src/main/fc/tasks.c

```diff
@@ -368,6 +370,11 @@ task_attribute_t task_attributes[TASK_COUNT] = {
 #endif
 
     [TASK_RX] = DEFINE_TASK("RX", NULL, rxUpdateCheck, taskUpdateRxMain, TASK_PERIOD_HZ(33), TASK_PRIORITY_HIGH), // If event-based scheduling doesn't work, fallback to periodic scheduling
+
+    #if defined(USE_HEADTRACKER) && defined(USE_SERIALTX)
+    // Run the SBus output task at 14ms intervals for standard rate
```

Comment:
So it is actually sbusTx task?

Reply:
No, it's the head tracker task. SBus is just a protocol we're going to output. SBus has nothing to output by itself, just as we don't have a CRSF or SBus task to deal with RX data.

Reply:
For SBus/CRSF rx, most processing is done in interrupt, so common RX task is enough.
SBus TX is more like telemetry, something must set rate of outgoing packets. Using headtracker task is fine only when it is only provider of SBUS Tx. Extensive redesign would be necessary for concurrent sources.